CONTRIBUTING.md

```diff
@@ -82,6 +82,20 @@ $ ninja install
 $ meson test
 ```
 
+## MacOS support (experimental)
+Most of the dependencies can be installed via [brew](brew.sh)
```

Comment:
Should be `[brew](https://brew.sh/)`, otherwise it will send you a file under this repo and 404.